mne/preprocessing/tests/test_ica.py

```diff
@@ -203,6 +204,7 @@ def test_ica_core():
         with warnings.catch_warnings(record=True):
             ica.fit(raw, picks=pcks, start=start, stop=stop)
             repr(ica)  # to test repr
+        'mag' in ica  # should now work
```

Comment:
in this context the return type actually doesn't matter, it just matters that it succeeds -- but I can add the correct answer to be extra careful